README.md

```diff
@@ -1,2 +1,66 @@
-# my-portfolio
-Building My Portfolio Website
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Porfolio Website
+
+> Building My Portfolio Website
+
+![screenshot](./app_screenshot.png)
+
+This is my portfolio website to showcase my skills and projects I have done as a software engineer. It is motivate by my Microverse program start on the 5th, July, 2021.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://henrykc24.github.io/my-portfolio/)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- Your Readme looks great. You did include the live link as suggested by the previous reviewer, the screenshot, other sections are also great. 
- [ ] [OPTIONAL] However, the "Getting Started" section could be improved with more descriptive content.

Reply:
I have greatly improved this. Thank you so much for this review. I am really learning a lot. :raised_hands: